docs/app/pages/components/components-sections/notifications/badge/badge.component.html

```diff
@@ -0,0 +1,158 @@
+                        <ux-checkbox [(value)]="badgeOverlap">badgeOverlap</ux-checkbox>
+                    </div>
+                    <div class="col-md-3 col-sm-6">
+                        <ux-checkbox [(value)]="badgeHidden">badgeHidden</ux-checkbox>
+                    </div>
+                </div>
+            </ux-accordion-panel>
+        </ux-accordion>
+    </div>
+</div>
+
+<hr />
+
+<p>
+    The <code>uxBadge</code> directive can be added to any element to provide additional information as a badge, such as
+    status text or unread count.
+</p>
+<p><strong>Note:</strong> the badge directive relies on the CSS overflow property being set to <code>visible</code> for
+    the subject element due to how the badge element attaches to it. If you want to use <code>hidden</code>, you will
+    need to add a wrapper element around the subject so it only affects the contents.</p>
+<p>
+    The appearance and behavior of the directive can be configured using the following properties:
+</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="badgeColor" type="string">
+        The color you want to use for the background color. This should be a string value that matches one of the colors
+        available in the <code>ColorService</code>.
+    </tr>
+    <tr uxd-api-property name="badgeBorderColor" type="string">
+        The color you want to use for the border of the badge. This should be a string value that matches one of
+        the colors available in the <code>ColorService</code>. If this value is not set the badge will have a transparent border.
+    </tr>
+    <tr uxd-api-property name="badgeValueLimit" type="number">
+        Sets a character or number limit on the content within the badge. If the content is a number and larger than
+        this value it will display this value with an appended <strong>+</strong>, otherwise the content will be treated
+        as text and use this value to truncate the number of characters displayed followed by an ellipsis
+        (<strong>&hellip;</strong>).
+    </tr>
+    <tr uxd-api-property name="badgeVerticalPosition" type="'above' | 'below'" defaultValue="'above'">
+        The vertical position of the badge in relation to its subject element.
+    </tr>
+    <tr uxd-api-property name="badgeHorizontalPosition" type="'before' | 'after'" defaultValue="'after'">
+        The horizontal position of the badge in relation to its subject
+        element.
+    </tr>
+    <tr uxd-api-property name="badgeOverlap" type="string" defaultValue="true">
+        This will determine whether the badge content will overlap its subject element. If overlap is
+        <code>false</code> the badge will start after the end of the subject element, otherwise
+        if it is set to <code>true</code> the badge horizontal center will be at the end of its subject element.
+    </tr>
+    <tr uxd-api-property name="badgeSize" type="string" defaultValue="'small' | 'medium' | 'large'">
```

Comment:
The type here should be `'small' | 'medium' | 'large'` and the default value `medium`